pkg/constants/constants.go

```diff
@@ -24,4 +26,10 @@ const (
 
 	ManagerName       = "kueue-manager"
 	JobControllerName = "kueue-job-controller"
+
+	// RequeueAfterDuration is the indicated time to wait before adding an item to the reconcile
+	// workqueue.
+	//
+	// TODO(#101)
```

Comment:
TODO what?

I suppose: "remove this"

Although it's useful to keep it long term as a batching mechanism.

So I would call the variable: "UpdatesBatchPeriod", similar to how we do it in the job controller:

https://github.com/kubernetes/kubernetes/blob/2b1b849d6a8bdeb7dc0807438cfd0ff2a9d752c1/pkg/controller/job/job_controller.go#L62

Reply:
I now, what you should say what you expect "to do" in the comment.